lib/log.c

```diff
@@ -83,7 +82,7 @@ void dolastlog (
 	newlog.ll_time = ll_time;
 	STRTCPY(newlog.ll_line, line);
 #if HAVE_LL_HOST
-	STRNCPY(newlog.ll_host, host);
+	STRNCPY(newlog.ll_host, "");
```

Comment:
STRNCPY() is semantically useful here, reminding that it's not a string.

Reply:
Not to belabor the point, but isn't that exactly what `memset` would covey as well? Anyway, I'm happy to concede the issue if you think that `STRNCPY` is more semantically meaningful here.

Reply:
> Not to belabor the point, but isn't that exactly what `memset` would covey as well?

Actually, yes.  I don't know why I said that.

> Anyway, I'm happy to concede the issue if you think that `STRNCPY` is more semantically meaningful here.

I still think STRNCPY() (or strncpy(3)) is more meaningful here, since memcpy(3) is a more generic call, also used on arbitrary data structures, while strncpy(3) conveys that it's a length-bound character array.